src/KDSoapClient/KDSoapUdpClient_p.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2019  Casper Meijn <casper@meijn.net>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef KDSOAPUDPCLIENT_P_H
+#define KDSOAPUDPCLIENT_P_H
+
+#include <QObject>
+#include <QUdpSocket>
+
```

Comment:
I had to include `KDSoapValue.h` (used for `KDSoap::SoapVersion` in line 40) and forward-declare `KDSoapUdpClient` to build it.